diff -Nur wxMaxima-0.6.2/wxmaxima.desktop wxMaxima-0.6.2.new/wxmaxima.desktop
--- wxMaxima-0.6.2/wxmaxima.desktop	2005-09-05 14:58:56.000000000 -0400
+++ wxMaxima-0.6.2.new/wxmaxima.desktop	2005-10-25 15:33:48.000000000 -0400
@@ -2,8 +2,8 @@
 Name=wxMaxima
 Comment=wxWidgets interface for CAS maxima
 Exec=wxmaxima
-Icon=maxima-new.png
+Icon=maximaicon.xpm
 Terminal=0
 Type=Application
 Encoding=UTF-8
-Categories=Application;Utility;X-Red-Hat-Base;X-Red-Hat-Base-Only;
\ No newline at end of file
+Categories=Application;Math;
