From: Gunter Königsmann <wxmaxima@physikbuch.de>
Description: lintian did claim that the desktopfile was missing a %f.
Author: Gunter Königsmann <wxmaxima@physikbuch.de>
Forwarded: Yes

--- a/data/wxMaxima.desktop.in
+++ b/data/wxMaxima.desktop.in
@@ -2,7 +2,7 @@
 Name=wxMaxima
 Comment=A graphical user interface that allows symbolic and numeric calculations using Maxima
 Comment[de]=Eine grafische Oberfläche für Maxima, ein Programm für Symbolische und Numerische Berechnungen
-Exec=@prefix@/bin/wxmaxima
+Exec=@prefix@/bin/wxmaxima %f
 Icon=wxmaxima
 Type=Application
 Categories=Education;Math;X-Red-Hat-Base;X-Red-Hat-Base-Only;
