From: Gunter Königsmann <wxmaxima@physikbuch.de>
Description: Removed the duplicate of the menu file.
Author: Gunter Königsmann <wxmaxima@physikbuch.de>
Forwarded: Yes

Index: wxmaxima-15.08.2/configure.ac
===================================================================
--- wxmaxima-15.08.2.orig/configure.ac
+++ wxmaxima-15.08.2/configure.ac
@@ -250,7 +250,6 @@ AC_CONFIG_FILES([
         wxmaxima.spec
         data/Info.plist
 	data/wxMaxima.desktop
-	data/wxmaxima.menu
 ])
 
 AC_OUTPUT
Index: wxmaxima-15.08.2/data/Makefile.am
===================================================================
--- wxmaxima-15.08.2.orig/data/Makefile.am
+++ wxmaxima-15.08.2/data/Makefile.am
@@ -12,10 +12,6 @@ dist_appdata_DATA = wxmaxima.appdata.xml
 appicondir = $(datarootdir)/pixmaps
 dist_appicon_DATA = wxmaxima.svg wxmaxima.png text-x-wxmaxima-batch.svg text-x-wxmathml.svg wxmaxima-16.xpm wxmaxima-32.xpm
 
-menudir = $(datarootdir)/menu
-dist_menu_DATA = wxmaxima.menu
-
-
 
 mandatadir = $(datadir)/man/man1
 dist_mandata_DATA = wxmaxima.1
Index: wxmaxima-15.08.2/data/wxmaxima.menu.in
===================================================================
--- wxmaxima-15.08.2.orig/data/wxmaxima.menu.in
+++ /dev/null
@@ -1,6 +0,0 @@
-?package(wxmaxima):\ needs="X11"\
-    section="Applications/Science/Mathematics"\ title="wxMaxima"\
-    longtitle="GUI for the computer algebra system Maxima"\
-    icon16x16="@prefix@/share/pixmaps/wxmaxima-16.xpm"\
-    icon32x32="@prefix@/share/pixmaps/wxmaxima-32.xpm"\
-    command="/usr/bin/wxmaxima"
