Index: wxmaxima-Version-22.05.0/CMakeLists.txt
===================================================================
--- wxmaxima-Version-22.05.0.orig/CMakeLists.txt
+++ wxmaxima-Version-22.05.0/CMakeLists.txt
@@ -221,7 +221,7 @@ add_subdirectory(test)
 
 add_custom_target(dist COMMAND ${CMAKE_MAKE_PROGRAM} package_source)
 
-install(FILES AUTHORS.md COPYING ChangeLog GPL.txt NEWS.md README README.md DESTINATION share/doc/wxmaxima)
+install(FILES AUTHORS.md COPYING GPL.txt NEWS.md README README.md DESTINATION share/doc/wxmaxima)
 
 # include wxWidgets DLLs on Windows on appveyor
 if(WIN32 AND APPVEYOR_BUILD)
