diff -Nur wxmaxima-0.7.1/src/wxMaxima.cpp wxmaxima-0.7.1.new/src/wxMaxima.cpp
--- wxmaxima-0.7.1/src/wxMaxima.cpp	2007-02-26 13:45:48.000000000 +0000
+++ wxmaxima-0.7.1.new/src/wxMaxima.cpp	2007-02-26 13:48:09.000000000 +0000
@@ -1039,7 +1039,7 @@
   wxString command, parameters;
   bool have_config = config->Read(wxT("maxima"), &command);
 
-  if (!have_config)
+  if (!have_config || (have_config && command.IsSameAs (wxT("1"))))
   {
     command = wxT("maxima");
     config->Write(wxT("maxima"), command);
