diff -Nur wxMaxima-0.7.0a/src/wxMaxima.cpp wxMaxima-0.7.0a.new/src/wxMaxima.cpp
--- wxMaxima-0.7.0a/src/wxMaxima.cpp	2006-09-27 14:18:54.000000000 +0200
+++ wxMaxima-0.7.0a.new/src/wxMaxima.cpp	2006-09-27 14:19:22.000000000 +0200
@@ -1120,13 +1120,16 @@
 
   wxString line;
   wxString docdir;
+  wxString prefix = wxEmptyString;
   wxString langsubdir;
 
   for (unsigned int i=0; i<output.GetCount(); i++)
   {
     line = output[i];
-    if (line.StartsWith(wxT("maxima-htmldir")))
-      docdir = line.Mid(15);
+    if (line.StartsWith(wxT("maxima-prefix")))
+      prefix = line.Mid(14);
+    else if (line.StartsWith(wxT("maxima-htmldir")))
+      docdir = prefix + line.Mid(15);
     else if (line.StartsWith(wxT("maxima-lang-subdir")))
     {
       langsubdir = line.Mid(19);
